repo-scripts/size-analysis/package.json

```diff
@@ -33,7 +33,7 @@
     "memfs": "3.4.7",
     "tmp": "0.2.1",
     "typescript": "4.7.4",
-    "terser": "5.15.1",
+    "terser": "5.16.1",
```

Comment:
Please merge the main branch into this branch AND into the base branch (`mila/BloomFilter`). That will fix this spurious diff.